app/views/records/_form.html.erb

```diff
@@ -17,6 +17,7 @@
                  :as => :key,
                  :locals => {:f => f}
       %>
+      <%= f.input :reviewed, :as => :boolean %>
```

Comment:
And probably give this a custom label; I think "Reviewed" will imply something entirely different than what we've got above